### PR TITLE
Show a `>` symbol in front of formatted number if less than 2 decimals

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/330?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#330** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/330?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#329** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/329?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#328** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/328?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#327** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/327?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#326** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/326?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#325** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/325?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#309** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/309?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#305** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/305?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#299** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/299?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by 0xBigBoss — COMMENTED** [review]
nice,  let's be sure to update the unit tests to include this new test case.

File: packages/app/utils/formatAmount.test.ts [hunk 1]
```diff
@@ -91,4 +91,12 @@ describe('Additional scenarios', () => {
   it('should not truncate if decimals match maxDecimals', () => {
     expect(formatAmount(123.45, 5, 2)).toBe('123.45')
   })
+
+  it('should handle when less than max decimals', () => {
+    expect(formatAmount(0.0045, 5, 1)).toBe('<0.0')
+  })
+
+  it('should handle when using default and less than two decimajs', () => {
```
Comment (0xBigBoss) [inline packages/app/utils/formatAmount.test.ts:99]: decimajs? 🤣

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Apr 22, 5:21 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/330).

**Pull request merged**